### PR TITLE
Issue2 farmer in malawi

### PR DESCRIPTION
**Pull request by TAlonglong**
#2

### CODE REVIEW

**Comment by TAlonglong** [comment]
Think this should be ok now.

**Comment by mortenwh** [comment]
There are two branches for issue #2 now: issue2_farmer_in_malawi and issue2_malawi_farmer :-D

Which is the correct one? :-P

**Comment by mortenwh** [comment]
I have deleted the branch issue2_farmer_in_malawi. That was something old that I had started and not the reason for your confusion :)

File: source/use_case_farmer_in_malawi.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+A farmer in Malawi receives daily forecasts from METs open data services. The forecast must arrive on a relevant platform and be presented in a way that makes it usable for the farming work.
+   
+Actors
+======
+
+.. 
+
+- Random user with limited experience (farmer in Malawi)
+- Data owner nwps (FOU-FD)
```
Comment (mortenwh) [inline source/use_case_farmer_in_malawi.rst]: isn't this SUV?

File: source/use_case_farmer_in_malawi.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+   .. uml::
+
+      @startuml Use case #2
+      !includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Container.puml
+
+      LAYOUT_LEFT_RIGHT
+
+      Person(malawi_farmer, "Farmer in Malawi")
+      System_Ext(search_interface, "Search Engine", "Google or met interface")
```
Comment (mortenwh) [inline source/use_case_farmer_in_malawi.rst]: I have a feeling the interface to the farmer should be by a dedicated (external) website and a mobile app. Then this should be connected as:

```
System(senda, "S-ENDA Find+Access/Central")
Rel(search_interface, senda, "Find and access weather prediction", "CSW and WMS")
```

**Comment by mortenwh** [comment]
@TAlonglong - please do the merging when you're done :)

File: source/use_case_farmer_in_malawi.rst (hunk 1 above)
Comment (TAlonglong) [reply source/use_case_farmer_in_malawi.rst]: Yes

File: source/use_case_farmer_in_malawi.rst (hunk 2 above)
Comment (TAlonglong) [reply source/use_case_farmer_in_malawi.rst]: ok

**Pull request merged**